pennylane/_qubit_device.py

```diff
@@ -25,25 +25,30 @@
 import numpy as np
 
 import pennylane as qml
-from pennylane import Device, DeviceError
-from pennylane.interfaces import set_shots
-from pennylane.math import multiply as qmlmul
-from pennylane.math import sum as qmlsum
+from pennylane import DeviceError
```

Comment:
are the changes here due to isort?

Reply:
To be honest I have no clue, I do not recall doing this intentionally 🤔 @eddddddy ?

Reply:
I'm guessing it's from the many merge commits on the branch from the other PRs. Of course the ordering here is inconsequential, but I can fix this so we don't have 20+ lines of noise.